sdk/core/Azure.Core/src/Shared/OperationInternal.cs

```diff
@@ -0,0 +1,207 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Pipeline;
+
+namespace Azure.Core
+{
+    internal class OperationInternal<TResult>
+    {
+        private const string RetryAfterHeaderName = "Retry-After";
+
+        private readonly object SetStateLock = new object();
+
+        private readonly IOperation<TResult> _operation;
+
+        private TResult _value;
+
+        private RequestFailedException _operationFailedException;
+
+        public OperationInternal(ClientDiagnostics clientDiagnostics, IOperation<TResult> operation)
+        {
+            _operation = operation;
+            OperationTypeName = operation.GetType().Name;
+            ClientDiagnostics = clientDiagnostics;
+            DefaultPollingInterval = TimeSpan.FromSeconds(1);
```

Comment:
Feels a bit too low.

Reply:
I'm keeping the same behavior we had in the old `OperationHelpers`. I believe that changing it would NOT be considered a breaking change, though. If we increase it (5 seconds maybe?), what should we do with clients that originally defaulted to 1 second?
- Keep using the default delay (behavior change: 1 second => 5 seconds).
- Manually set the delay to 1 second to overwrite the new default delay (preserve behavior).

I believe the first approach should be fine. Clients concerned about the delay time already set their times manually (Key Vault (2), Synapse (5), Remote Rendering (10)). Defaulting to me sounds like "use whatever delay the common implementation thinks it's best", which may be subject to changes.

Reply:
Didn't realize the existing one is that low.